README.md

```diff
@@ -641,4 +641,63 @@ Feature: Gestionar los habilidades para los usuarios
         Given el servicio de gestión de habilidades "skill-controller" sin autenticación válida 
         When se hace una petición POST al endpoint /skills con un nombre de habilidad NO existente 
         Then el servicio responde un código HTTP 401 
-```
\ No newline at end of file
+```
+
+### 7. Configurar Integración Continua (CI)
+
+Para iniciar esta actividad, crea una nueva rama de tu proyecto.
+
+Para crear la configuración del workflow de GitHub actions, vamos a crear un archivo maven.yml en el directorio `.github/workflows` que realice los siguientes steps cuando creamos o actualizamos un Pull Request:
+* Configuración de java
+* Construye el proyecto con Maven
+
+    1. Para esto puedes usar la plantilla que genera gitHub Action
+    * Ingresa a tu repositorio git desde la web 
+    * Seleccione la pestaña Actions
+![branch rules](https://github.com/testing-community/rest-assured-workshop/blob/main/media/gitHubActiosOption.png)
+    
+* Acciona New workflow
+* Acciona Configurar
+* Selecciona la opción Maven Java Option, te presentara un nuevo archivo .yaml con una configuración estándar.
+![branch rules](https://github.com/testing-community/rest-assured-workshop/blob/main/media/workFlowJavaMavenOption.png)
```

Comment:
Use path but to the folder you already have to not depend on url

Reply:
done